pyflakes/checker.py

```diff
@@ -2120,6 +2131,7 @@ def IMPORT(self, node):
             else:
                 name = alias.asname or alias.name
                 importation = Importation(name, node, alias.name)
+            importation.during_type_checking = self._in_type_checking
```

Comment:
this should be part of the construction

Reply:
Will do! There is quite a hierarchy of classes that will touch but it makes sense I suppose.